packages/azure-functions/ProcessMaliciousUploads/index.mjs

```diff
@@ -0,0 +1,89 @@
+import { blobStorageConnector } from '@defra/bng-connectors-lib'
+
+const MALWARE_SCANNNING_RESULT_TAG_NAME = 'Malware Scanning scan result'
+const MALWARE_SCANNNING_RESULT_TAG_VALUE = 'Malicious'
+
+const containerName = 'customer-uploads'
+const tags = `"${MALWARE_SCANNNING_RESULT_TAG_NAME}" = '${MALWARE_SCANNNING_RESULT_TAG_VALUE}'`
+const config = {
+  containerName,
+  tags
+}
+
+let itemProcessingTimeout
+let processingTimeout
+
+export default async function (context, _timer) {
+  context.log('Processing malicious uploads')
+  let pauseProcessing = false
+  // If all malicious ulploads are not processed within a configurable period of time,
+  // pause processing unti the next scheduled invocation of this function.
+  const timeoutId = setTimeout(() => { pauseProcessing = true }, getProcessingTimeout(context))
+  for await (const maliciousUpload of await blobStorageConnector.findBlobsInContainerByTags(config)) {
```

Comment:
could we extract `await blobStorageConnector.findBlobsInContainerByTags(config)` to const just to simplify line 22?

Reply:
Yes but it is not vital.